### DISCUSSION TITLE
Embedded video

### DISCUSSION THREAD

**Discussion by daxime** [zim-desktop-wiki/zim-desktop-wiki#1828] [Ideas]
Hi,
It will be nice to embed video files like ZimWiki do for images. If you dont want to add a video player, maybe you can make it launch the default player of the system.

**Comment by quesada** [comment]
+1

**Comment by introt** [comment]
Duplicate of #506, which I've converted to a Discussion due to inactivity; please continue discussion over at #1839 and upvote it for visibility
